test/test_eos_ideal.cpp

```diff
@@ -32,7 +32,7 @@
 using singularity::IdealGas;
 using EOS = singularity::Variant<IdealGas>;
 
-SCENARIO("Ideal gas entropy", "[IdealGas][Entropy]") {
+SCENARIO("Ideal gas entropy", "[IdealGas][Entropy][FreeEnergy]") {
```

Comment:
FreeEnergy can be Gibbs or Helmholtz. Maybe GFreeEnergy?

Reply:
Changed to `GibbsFreeEnergy` I don't see any reason not to be verbose here.